### PR TITLE
feat(proxy): add mds_direct plugin for direct access of mds cloud.

### PR DESCRIPTION
**Pull request by antmat**
@noxiouz @3Hren  PTAL

### CODE REVIEW

File: cocaine/proxy/mds_direct.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            allow_ipv6=True,
+            request_timeout=timeout)
+
+        try:
+            # NOTE: we can do it in a streaming way
+            resp = yield self.srw_httpclient.fetch(srw_request)
+            body = resp.buffer.read(None)
+            lines = body.split("\n")
+            backends = []
+            for line in lines:
```
Comment (noxiouz) [inline cocaine/proxy/mds_direct.py]:
```python
backends = [line.split()[0] for line in lines if line]
```

File: cocaine/proxy/mds_direct.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                if line:
+                    backends.append(line.split()[0])
+
+            app = Service(name, endpoints=[(backends[0], 10071)], timeout=RESOLVE_TIMEOUT)
```
Comment (noxiouz) [inline cocaine/proxy/mds_direct.py]: Why only backends[0]?

Comment (noxiouz) [inline cocaine/proxy/mds_direct.py]: 10071?

Comment (antmat) [inline cocaine/proxy/mds_direct.py]: oops, will move inside conf

File: cocaine/proxy/proxy.py [hunk 3]
```diff
@@ -740,6 +744,7 @@ def stop_condition(body):
                                  message, proxy_error_headers())
 
             except Exception as err:
+                request.logger.exception(err)
```
Comment (noxiouz) [inline cocaine/proxy/proxy.py]: https://docs.python.org/2/library/logging.html#logging.Logger.exception

```python
request.logger.exception("%s: %s", app.id, err)
```
And drop the next line

**Comment by noxiouz** [comment]
LGTM

**Pull request merged**